smart_open/s3.py

```diff
@@ -112,6 +114,7 @@ def open(
         fileobj = SeekableBufferedInputBase(
             bucket_id,
             key_id,
+            version_id,
```

Comment:
Pass keyword parameters by name.

```suggestion
            version_id=version_id,
```